dotnet/xml/Microsoft.ML/VisionCatalog.xml

```diff
@@ -79,6 +79,14 @@
         </summary>
         <returns>To be added.</returns>
         <remarks>To be added.</remarks>
+        <example>
+          <format type="text/markdown">
+            <![CDATA[
+             [!code-csharp[ImageClassification](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/ImageClassification/ResnetV2101TransferLearningTrainTestSplit.cs)]
```

Comment:
Would we want the default sample to come first?

Reply:
@codemzs ?

Reply:
Probably. 

---
In reply to: [343390917](https://github.com/dotnet/ml-api-docs/pull/116#discussion_r343390917) [](ancestors = 343390917)

Reply:
Looks good. It should be a cref though. Have a look at: https://docs.microsoft.com/en-us/dotnet/api/microsoft.ml.standardtrainerscatalog?view=ml-dotnet as an example